src/kernels/composable_kernel/include/tensor_operation/gridwise_gemm_xdlops_fp16_bfp16.hpp

```diff
@@ -365,6 +357,7 @@ struct GridwiseGemmTransposedANormalBNormalCXdlopsFp16Bfp16_v1
             constexpr index_t BlkSize = OutputLayout.GetBlkSize();
             constexpr index_t NumBlks = OutputLayout.GetNumBlks();
 
+#pragma unroll
```

Comment:
@zjing14 This pragma unroll is crucial to performance. Perhaps it's worthwhile to add comment to explain its importance?

Reply:
Added